src/Hud/UI/Renderers/TextureRenderer.cs

```diff
@@ -18,6 +21,10 @@ public TextureRenderer(Device device)
             this.device = device;
             sprite = new Sprite(device);
             textures = new Dictionary<string, Texture>();
+            System.Timers.Timer t = new System.Timers.Timer(1000);
```

Comment:
why are we using timers? Can you remove timer, and put them in your plugin rather than in the core?